var/spack/repos/builtin/packages/gaussian/package.py

```diff
@@ -18,6 +18,19 @@ class Gaussian(Package):
     version('16-B.01', sha256='0b2cf60aa85d2c8c8e7547446e60e8e8cb67eec20e5f13c4a3e4e7616dcdf122')
     version('09-D.01', sha256='ef14885b5e334b6ec44a93bfd7225c634247dc946416af3087ab055bf05f54cd')
 
+    depends_on('tcsh')
```

Comment:
Nice catch! Your patch is very clean too. Just one silly question, you tested this on your stripped down system?
I just want to be sure it works on a production system.

Reply:
I tested the build process on the system we use to build packages. I haven't actually tested the whole functionality of the software. It should be fine, since all we're really doing here is changing which csh interpreter is called; the spack one vs the system one.